batch7crud-roll7/src/main/webapp/static/css/layout.css

```diff
@@ -0,0 +1,158 @@
+.header {
+    width: 100%;
+    background: #3e9cca;
+}
+
+.header-text {
+    margin: 0 auto;
+    padding-top: 50px;
+    padding-bottom: 50px;
+    color: #fff;
+    text-align: center;
+    font-size: 32px;
+}
+
+.mainbody {
+    width: 100%;
+    background: #eee;
+    height: 500px;
+}
+
+.login-container {
+    width: 300px;
+    margin: 0px auto;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unit for this zero value [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Azero-units)